libdebug/state/resume_context.py

```diff
@@ -6,45 +6,15 @@
 
 from __future__ import annotations
 
-from enum import Enum
-
-
-class ResumeStatus(Enum):
-    """Enum representing if the process should be resumed or not after a wait.
-
-    Attributes:
-        UNDECIDED (int): The stop reason is not managed. In this case, the decision is left to the user
-        according to what specified by the force_resume parameter of the debugger.
-        NOT_RESUME (int): The process should not be resumed.
-        RESUME (int): The process should be resumed.
-    """
-
-    UNDECIDED: int = 0
-    NOT_RESUME: int = 1
-    RESUME: int = 2
-
 
 class ResumeContext:
     """A class representing the context of the resume decision."""
 
     def __init__(self: ResumeContext) -> None:
         """Initializes the ResumeContext."""
-        self._resume: ResumeStatus = ResumeStatus.UNDECIDED
+        self.resume: bool = True
```

Comment:
These public attributes should be documented or made private

Reply:
It is not a debugger attribute. It will be documented in the full documentation but is not something the user will use